### DISCUSSION TITLE
Lesson 9 - Problem with json

### DISCUSSION THREAD

**Discussion by Loupsteb** [Cyfrin/foundry-full-course-cu#1218] [Q&A]
hi everyone, i follow the course Foundry Fundamentals and i have a problem right now.
I've make the foundry command for compil the simpleStorage contract but on my SimpleStorage.json document the document isn't format.
It's the first time i have this type of problem, i share with you a screenshot of what is looking.
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/52604984/ee97ff9a-2d08-419f-bc91-cd93e84d9a96)

**Comment by Jsyn00** [comment]
Hello! I have the same problem with the format in this file.

Do we need some kind of extension? Any recomendation? 

Thanks

**Reply by Loupsteb** [reply]
I solve this problem with a simple right click and format document.
Normaly it's will be work for you to

**Reply by Jsyn00** [reply]
Yes, it works!!! Thanks

**Reply by Loupsteb** [reply]
You're welcome
